frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/storage-and-nodes-step.tsx

```diff
@@ -82,13 +197,27 @@ export const StorageAndNodes: React.FC<StorageAndNodesProps> = ({ state, dispatc
   const setNodes = (filteredData: NodeKind[]) =>
     dispatch({ type: 'setNodes', value: filteredData });
 
+  const filterSC = ({ resource }): boolean => {
+    const noProvSC = filterSCWithNoProv(resource);
+    if (stretchClusterChecked && noProvSC && !nodesError && nodesData.length && nodesLoaded) {
+      return isArbiterSC(resource, pvData, nodesData);
```

Comment:
Why not this ?
```suggestion
      return isArbiterSC(resource, scNodeNames, nodesData);
```

Reply:
IsAbiterSC is testing for all the storage classes and not the selected one. The scNodeName is for selected storageclass.

Reply:
Okay, I see!